### ISSUE TITLE
Multipass failing to launch

### ISSUE THREAD

**Issue by s-makin** [canonical/multipass#3680]
**Describe the bug**
For the past few hours, trying to launch an instance gives the `launch failed: Remote "" is unknown or unreachable.` response. I am trying to test a tutorial that uses Multipass as the VM. When I last tested the tutorial about a week ago, I didn't get this error.

I have been periodically retrying all morning and did manage to create one instance (which works just fine). I have tried testing this with other launch commands and it's giving the same response, so doesn't seem to depend on release. 

**To Reproduce**
Commands tried:
1. `multipass launch noble --name tutorial`
1. `multipass launch noble --name test --disk 512M --memory 128M`
1. `multipass launch xenial`
1. `multipass launch jammy`

Response to all is the same:
1. `launch failed: Remote "" is unknown or unreachable.`

**Expected behavior**
To launch the various instances given the image and conditions specified.

**Logs**
[multipasslog.txt](https://github.com/user-attachments/files/17044001/multipasslog.txt)

**Additional info**
 - OS: [e.g. macOS 10.15]
- `multipass version`
multipass 1.14.0
multipassd 1.14.0

- `multipass info`
Name:           tutorial
State:          Running
Snapshots:      0
IPv4:           10.80.65.212
Release:        Ubuntu 24.04.1 LTS
Image hash:     78547d336e4c (Ubuntu 24.04 LTS)
CPU(s):         1
Load:           0.00 0.00 0.00
Disk usage:     2.0GiB out of 4.8GiB
Memory usage:   350.4MiB out of 956.1MiB
Mounts:         --

- `multipass get local.driver`
qemu

**Additional context**
The URLs the log says can't be fetched are accessible when I paste the link into the browser.

**Comment by andrei-toterman** [comment]
Hey, @s-makin! You could try running `multipass find --force-update` and then try launching again. If that does not work, try restarting the Multipass service with `sudo snap restart multipass`.

**Comment by s-makin** [comment]
> Hey, @s-makin! You could try running `multipass find --force-update` and then try launching again. If that does not work, try restarting the Multipass service with `sudo snap restart multipass`.

Thanks for the reply @andrei-toterman, running a force update gives me this response:
`find failed: failed to download from 'https://cdimage.ubuntu.com/ubuntu-core/16/stable/current/ubuntu-core-16-amd64.img.xz': Operation canceled`

Restarting multipass also didn't work - this gave the same `launch failed: Remote "" is unknown or unreachable.` error as before :/

**Comment by s-makin** [comment]
I tried manually downloading the image in the link, and it worked without issues, so I'm not sure why Multipass can't find it

**Comment by andrei-toterman** [comment]
Huh, that output and those logs indicate that this is a duplicate of #3662, which is weird. This will need some further investigations...

**Comment by s-makin** [comment]
> Huh, that output and those logs indicate that this is a duplicate of #3662, which is weird. This will need some further investigations...

No problem, let me what you need from me :)

**Comment by ricab** [comment]
Hey @s-makin, to try to unblock you (and if you don't have other instances you need to keep) you can try `snap remove --purge multipass` and then `snap install multipass` again. 

I am sorry I don't have a better answer for you, but it is unclear at this point what is going on. Not easy to investigate without a way to reproduce. I am going to mark this one as duplicate, but please chime in on the other one if you have any further clue or other input! Thanks.

**Comment by ricab** [comment]
Duplicate of #3662
